Ft create user profile page 163380504

Jesse-efe opened the pull request:
#### What does this PR do?
Create a user profile page
#### Description of Task to be completed?
Page shows user vote history
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163380504
#### Screenshots
![one](https://user-images.githubusercontent.com/35180482/51613713-60831d80-1f24-11e9-98f9-ebdad953ce33.JPG)

#### Questions:
N/A

Jesse-efe committed:
Create the home page Finishes [Delivers #163312850]

Jesse-efe committed:
Created sign in page [Delivers #16329572]

Jesse-efe committed:
Create user sign up page [Delivers #163295274]

Jesse-efe committed:
Create Password reset page [Delivers #163296269]

Jesse-efe committed:
feature(createUserPageThatShowsAllPoliticalParties):

- page has a nav bar
- page has cards for each party
- page has a footer

[delivers #163351595]

Jesse-efe committed:
feat(design/template): make changes to UI

-add a navbar to sign-up, sign-in and reset password pages

[delivers #163384443]

Jesse-efe committed:
feat(design/template): make changes to nav bar button

-change the color of the text in the button

[delivers #163418388]

Jesse-efe committed:
feat(design/template): change text in sign-up page submit button

-change the text from sign-in to sign-up

[delivers #163419537]

Jesse-efe committed:
ch(design/template): remove commented out code in css files

[finishes #163419946]

Jesse-efe committed:
feat(design/template): change color of buttons in cards

-change the color of buttons in cards in parties.html file

[delivers #163420400]

Jesse-efe committed:
feat(design/template): change color of button in home page

-change the color of call to action button

[delivers #163420702]

Jesse-efe committed:
feat(design/template): create a user profile page

-page shows user vote history
-page has a menu with which user can navigate to other pages

[delivers #163380504]

megame24 commented:
How come you have commits dating back to as far as four days ago, this PR should only contain its own commits. Please refer to the resource on Git workflow I shared with you to understand how your Git workflow should be set up. If the resource is not of much help, please reach out to your fellow learners via the group channel.

Jesse-efe committed:
feat(design/template): make user profile page responsive

-hide the menu on small devices
-add a button to show and hide the menu

[delivers #163445712]

Jesse-efe commented:
Ok I will. Thanks
